server/server.py

```diff
@@ -45,18 +45,15 @@ def recommend(self, video_id):
 def index():
     return render_template('index.html')
 
-@app.route('/song', methods=['GET'])
+@app.route('/search', methods=['GET'])
```

Comment:
where are you sending the response to the front-end?

Reply:
Check out search and recommend endpoints, in search I have sent the video_id and title of the result to the front end, and in recommend I have sent the list of songs with title and video_id to the front end. The play endpoint however, just plays the song, it doesn’t return anything.